javascript/javascript1/week3/homeworkweek03.js

```diff
@@ -0,0 +1,113 @@
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+names.splice(1,1);
```

Comment:
Good job!!
Could you try to write a function which can remove any name from the array.

Reply:
do you mean random ?

Reply:
Yes, now you can remove only the name 'Ahmad', write a function which can remove any name within the array.